akka-cluster-sharding/src/main/mima-filters/2.5.17.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# #23751 warn if handOffStopMessage not handled
```

Comment:
This should be last-version`.backwards.excludes`, so in this case 2.5.16 (https://github.com/akka/akka/blob/master/CONTRIBUTING.md#binary-compatibility)

Reply:
Oops! Thanks.

Reply:
@salehkhazaei and now it's 2.5.18

Reply:
Thanks.